app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapView.jsx

```diff
@@ -594,43 +595,54 @@ class SamplesHeatmapView extends React.Component {
           taxonFilterState={this.state.taxonFilterState}
           thresholdFilters={this.state.selectedOptions.thresholdFilters}
           sampleSortType={this.state.selectedOptions.sampleSortType}
+          fullScreen={this.state.hideFilters}
         />
       </ErrorBoundary>
     );
   }
 
+  toggleDisplayFilters = () => {
+    this.setState(prevState => ({ hideFilters: !prevState.hideFilters }));
+  };
+
   render() {
     return (
       <div className={cs.heatmap}>
-        <NarrowContainer>
-          <SamplesHeatmapHeader
-            sampleIds={this.state.sampleIds}
-            data={this.state.data}
-            onDownloadSvg={this.handleDownloadSvg}
-            onDownloadPng={this.handleDownloadPng}
-            onDownloadCsv={this.handleDownloadCsv}
-            onShareClick={this.handleShareClick}
-            onSaveClick={this.handleSaveClick}
+        {!this.state.hideFilters && (
+          <div>
+            <NarrowContainer>
+              <SamplesHeatmapHeader
+                sampleIds={this.state.sampleIds}
+                data={this.state.data}
+                onDownloadSvg={this.handleDownloadSvg}
+                onDownloadPng={this.handleDownloadPng}
+                onDownloadCsv={this.handleDownloadCsv}
+                onShareClick={this.handleShareClick}
+                onSaveClick={this.handleSaveClick}
+              />
+            </NarrowContainer>
+            <NarrowContainer>
+              <SamplesHeatmapControls
+                options={this.getControlOptions()}
+                selectedOptions={this.state.selectedOptions}
+                onSelectedOptionsChange={this.handleSelectedOptionsChange}
+                loading={this.state.loading}
+                data={this.state.data}
+              />
+            </NarrowContainer>
+          </div>
+        )}
+        {this.state.hideFilters && <Divider style="shadow" />}
+        <div
+          className={cs.filterToggleContainer}
+          onClick={this.toggleDisplayFilters}
```

Comment:
I forgot to say that this action should log to analytics so we can see how many users click on it over time.